### DISCUSSION TITLE
The nuclei process will stop for no reason

### DISCUSSION THREAD

**Discussion by ep113** [projectdiscovery/nuclei#4873] [Q&A]
[0:10:00] | Templates: 2443 | Hosts: 943 | RPS: 104 | Matched: 0 | Errors: 36798 | Requests: 62833/4412297 (1%)
[0:20:00] | Templates: 2443 | Hosts: 943 | RPS: 123 | Matched: 0 | Errors: 48480 | Requests: 148720/4412297 (3%)
[0:24:56] | Templates: 2443 | Hosts: 943 | RPS: 103 | Matched: 0 | Errors: 49644 | Requests: 155536/4412297 (3%)

Then exit directly to the command line

I am using the Kali operating system. What is the reason for this

**Comment by GeorginaReeder** [comment]
Hi there @ep113 ! What kind of scan are you trying to run when you're getting this output? Is it a full scan, or something else? The more detail you can give us, the better.

**Reply by ep113** [reply]
Hellow The following are the scan command parameters I used
nuclei -l url.txt -c 100 -s high,critical -timeout 5 -mhe 10 -me result -o result.txt -stats -si 600

**Reply by ep113** [reply]
system kernel：
Linux iZj6cbstuihgwfsdfjgms0gruZ 6.0.0-kali6-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.0.12-1kali1 (2022-12-19) x86_64 GNU/Linux

**Reply by ep113** [reply]
After replacing the Windows operating system, the scanning progress was 83% and the process ended

**Reply by tarunKoyalwar** [reply]
@ep113, currently stats counter in nuclei is not 100% correct and may exit early due to dynamic number of payloads and `stop-at-first-match` , we are currently in the process of fixing it, meanwhile can you confirm the exit code of command ? or did it print any more details before exiting `Killed` or `Error Code 137` ? the info you have shared is not enough to determine the issue

**Reply by ep113** [reply]
./nuclei -l /root/it/it1.txt -c 100 -s high,critical -timeout 5 -stats -si 600
[INF] Current nuclei version: v3.1.10 (latest)
[INF] Current nuclei-templates version: v9.7.8 (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] New templates added in latest release: 126
[INF] Templates loaded for current scan: 2443
[INF] Executing 2447 signed templates from projectdiscovery/nuclei-templates
[INF] Targets loaded for current scan: 943
[INF] Templates clustered: 165 (Reduced 174455 Requests)
[INF] Skipped x.x.x.x:443 from target list as found unresponsive 32 times
[INF] Using Interactsh Server: oast.pro
[INF] Skipped x.x.x.x:7443 from target list as found unresponsive 31 times
[INF] Skipped x.x.x.x:443 from target list as found unresponsive 34 times
......
[0:10:00] | Templates: 2443 | Hosts: 943 | RPS: 104 | Matched: 0 | Errors: 36798 | Requests: 62833/4412297 (1%)
......
[0:20:00] | Templates: 2443 | Hosts: 943 | RPS: 123 | Matched: 0 | Errors: 48480 | Requests: 148720/4412297 (3%)
[INF] Skipped x.x.x.x:7443 from target list as found unresponsive 31 times
[INF] Skipped x.x.x.x:443 from target list as found unresponsive 34 times
[0:24:56] | Templates: 2443 | Hosts: 943 | RPS: 103 | Matched: 0 | Errors: 49644 | Requests: 155536/4412297 (3%)
(root@xxxxxx)-[~/go/bin]

Thanks for your reply！
The above is the complete output of my running program, and I have filtered out sensitive information.
It is precisely because there is no error message output that I am unsure, so I have no way to start
Or you can tell me which places have detailed logs recorded locally to further verify the detailed error logs

**Comment by Yahoootestbyme** [comment]
I am also facing the same issue.. How you solved it?

**Comment by the5orcerer** [comment]
Facing the same issue!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
